src/components/ArtifactCard.vue

```diff
@@ -43,10 +43,10 @@
   import { computed } from 'vue'
   import FormattedDate from '@/components/FormattedDate.vue'
   import { localization } from '@/localization'
-  import { Artifact } from '@/models'
+  import { Artifact, ArtifactCollection } from '@/models'
 
   const props = defineProps<{
-    artifact: Artifact,
+    artifact: Artifact | ArtifactCollection,
```

Comment:
With the intro of the `kind` field, `Artifact` and `ArtifactCollection` aren't exactly equivalent; this allows them to be passed interchangeably